projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order.config.ts

```diff
@@ -626,7 +624,6 @@ export const tabbingOrderConfig: TabbingOrderConfig = {
       type: TabbingOrderTypes.CX_MEDIA,
     },
     { value: 'Show reviews', type: TabbingOrderTypes.LINK },
-    { value: 'quantity', type: TabbingOrderTypes.ITEM_COUNTER },
```

Comment:
Can we also remove `ITEM_COUNTER` from `TabbingOrderTypes` if it is no longer used?